config/routes.rb

```diff
@@ -58,6 +58,10 @@
 
   resources :annotations, only: [:new, :create, :edit, :update, :destroy]
 
```

Comment:
If you add the index action to the AnnotationsController, then the annotation_controller should have all seven RESTful actions and you can just write 
`resources :annotations`
and delete the next lines.
What I am wondering is: Why is there a `show` action in the `AnnotationController` but no route for it? Is it unused?

Reply:
To be honest, I don't know why we have a `show` method there. Things work just fine without it. Will investigate in future PR when I write cypress tests for the annotations.